src/Templates/src/templates/maui-mobile/Platforms/Android/MainActivity.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace MauiApp._1;
 
-[Activity(Theme = "@style/Maui.SplashTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation | ConfigChanges.UiMode | ConfigChanges.ScreenLayout | ConfigChanges.SmallestScreenSize | ConfigChanges.Density)]
+[Activity(Theme = "@style/Maui.SplashTheme", MainLauncher = true, LaunchMode = LaunchMode.SingleTop, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation | ConfigChanges.UiMode | ConfigChanges.ScreenLayout | ConfigChanges.SmallestScreenSize | ConfigChanges.Density)]
```

Comment:
Do we handle `OnNewIntent()`? I was reading: https://medium.com/android-news/android-activity-launch-mode-e0df1aa72242

If something uses this `Bundle`, we would need to handle `OnNewIntent()` and update it?

https://github.com/dotnet/maui/blob/dab9920b3fb9c0c2af66368cfab3dac7ef6cc630/src/Core/src/ActivationState.cs#L12

Otherwise, I wouldn't recommend updating the `LaunchMode` in the template?

Reply:
>we would need to handle OnNewIntent() and update it?

I'm not sure what you mean by "need to" handle. I don't think there's anything we necessarily need to account for. We could definitely add some helpful hooks to users, but, users can handle `OnNewIntent` themselves through `LifeCycle` events or by overriding `OnNewIntent` on the Activity. Anything that has to be handled by `OnNewIntent` will be something the user needs to be aware of for their given scenario. 

The problem right now is that our template isn't setup to handle multiple created activities, so, the alternative here is that the app will crash. Users have to either Override CreateWindow or they have to modify the LaunchMode

The point of the template change really is if users are integrating with 3rd party libraries like Shinie https://github.com/dotnet/maui/issues/18692#issuecomment-2016524388 those will now just work more seamlessly.

Reply:
Is the Bundle above on ActivationState used for anything? You would need to override OnNewIntent to replace it with the new one.

Reply:
AFAIK we don't use it anywhere.

It's just provided for the users benefit if they want to interact with it during the CreateWindow call. 

Ideally we'd maybe have some sort of event on Window like "ActivationChanged" or similar.

From searching around though it seems mixed whether you'll use setIntent or not.

https://stackoverflow.com/a/7749347

The understanding I have about `OnNewIntent` is that the user has to decide if they want that to represent the new world order or not. Sticking with the old intent is just fine if they don't care about reacting to the new intent when the app is backgrounded and the foregrounded.